rs/execution_environment/benches/lib/src/wat.rs

```diff
@@ -31,7 +32,13 @@ pub enum Module {
 impl Module {
     /// Render a complete WAT module for a system call executing in a loop, with params and result.
     #[allow(clippy::wrong_self_convention)]
-    pub fn from_ic0<N, P>(&self, name: N, params: P, result: Result) -> String
+    pub fn from_ic0<N, P>(
+        &self,
+        name: N,
+        params: P,
+        result: Result,
+        wasm64_enabled: Wasm64,
```

Comment:
NIT: since this variable can be both enabled and disabled I'd rename it to `wasm64_status` or `wasm64_feature` or `wasm64_flag`.

Reply:
thanks, done.